CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html),
 and is generated by [Changie](https://github.com/miniscruff/changie).
 
+## Unreleased
+
+### Bug Fixes
+
+- [runtime] Invalid YAML files no longer cause package detection to fail. [#665](https://github.com/pulumi/pulumi-yaml/pull/665)
+
```

Comment:
This repo uses changie for the changelog. Don't edit the changelog directly. Instead use `changie new` per https://github.com/pulumi/pulumi-yaml/blob/main/CONTRIBUTING.md#changelog